python/dask_cudf/dask_cudf/backends.py

```diff
@@ -481,6 +481,31 @@ def sizeof_cudf_series_index(obj):
     return obj.memory_usage()
 
 
+# TODO: Remove try/except when cudf is pinned to dask>=2023.10.0
+try:
+    from dask.dataframe.dispatch import partd_encode_dispatch
+
+    @partd_encode_dispatch.register(cudf.DataFrame)
+    def _simple_cudf_encode(_):
+        # Basic pickle-based encoding for a partd k-v store
+        import pickle
+        from functools import partial
+
+        import partd
```

Comment:
Can we add `partd` to our package requirements and conda recipes?

Reply:
I think it comes in transitively through dask? Which is a dependency of dask-cudf.

Reply:
Right exactly, [dask depends on partd](https://github.com/dask/dask/blob/91098a634967dfb44d67da54b1394d730f75e87d/pyproject.toml#L37). I think it should be safe for us to let dask worry about the partd dependency. If dask suddenly stops using partd for `shuffle="disk"`, it will also stop using `partd_encode_dispatch`.